src/main/java/org/broadinstitute/hellbender/tools/walkers/fasta/FastaReferenceMaker.java

```diff
@@ -82,7 +83,10 @@
     public void onTraversalStart() {
         final Path path = IOUtils.getPath(output);
         try {
-            writer = new FastaReferenceWriter(path, basesPerLine, true, true);
```

Comment:
Is the behavior previously controlled by the two `true` booleans here being preserved in the new `FastaReferenceWriterBuilder` call?

Reply:
Looks like these were `makeFai` and `makeDict`, both of which are `true` by default in the new builder.